CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+All development related to accessibility is currently happening in the [Stardust react](https://github.com/stardust-ui/react) repository. To contribute, create a pull request in that repository.
+
+# ARIA conformance
+In general, components need to conform to the [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices-1.1/). These practices are evaluated to achieve best possible user experience with supported browser/screen reader combinations.
+
+# Components and Behaviors
+There are two areas that come together to achieve accessibility:
+* Components - implemented for each supported framwork (currently React), when rendered, need to be [semantically correct](https://en.wikipedia.org/wiki/Semantic_HTML)
+* Behaviors - framework independent, intended to add ARIA roles, ARIA attributes and keyboard handling on top of the components. In the future, behaviors development will move from the [Stardust react](https://github.com/stardust-ui/react) repo to [Stardust accessibility](https://github.com/stardust-ui/accessibility) repo.
```

Comment:
I'd mention here that behaviors generate aria-* attributes based on component state, as this is quite an important thing. e.g. if a component is _expanded_, behavior will change `aria-expanded` attribute to `true` and it should be `false` otherwise